packages/debug-test/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "debug-test",
+	"version": "0.0.0",
+	"private": "true",
+	"scripts": {
+		"debugtest": "jest",
+		"lint": "rnw-scripts lint",
+		"lint:fix": "rnw-scripts lint:fix"
+	},
+	"devDependencies": {
+		"@react-native-windows/automation": "^0.1.73",
+		"@react-native-windows/fs": "^0.0.0-canary.3",
+		"@rnw-scripts/eslint-config": "1.1.13",
+		"@types/jest": "^26.0.20",
+		"@typescript-eslint/eslint-plugin": "^5.20.0",
+		"@typescript-eslint/parser": "^5.20.0",
+		"lodash": "^4.17.15",
+		"ts-jest": "^26.3.0",
```

Comment:
>ts-jest [](http://example.com/codeflow?start=3&length=7)

Curious, and echoing the concern raised by Nick .. Should  we avoid taking dependency on ts-jest (at least for hygeine and consistancy across the repo) ? There are other test packages which are using typescipt ..